### PR TITLE
IDE-4317 give a fixed name for gradle watch launch configuration

### PR DESCRIPTION
**Pull request by jtydhr88**
a long task name will cause error during buildship persisting configuration

### CODE REVIEW

**Comment by jtydhr88** [comment]
@Camork can you open one buildship issue and make some contribution on it? thanks @lily94299570 to find this problem as well

File: tools/plugins/com.liferay.ide.gradle.core/src/com/liferay/ide/gradle/core/GradleUtil.java [hunk 1]
```diff
@@ -196,6 +199,33 @@ public static void runGradleTask(IProject project, String[] tasks, String[] argu
 		launchConfigurationWC.launch(ILaunchManager.RUN_MODE, monitor);
 	}
 
+	public static void runGradleTask(
```
Comment (gamerson) [inline tools/plugins/com.liferay.ide.gradle.core/src/com/liferay/ide/gradle/core/GradleUtil.java]: This method is very very close to another method in this same class.  can you merge those two methods into one and support both functionality?

File: tools/plugins/com.liferay.ide.gradle.ui/src/com/liferay/ide/gradle/ui/RemoveOldWatchConsoleListener.java [hunk 2]
```diff
@@ -50,7 +52,7 @@ public void consolesAdded(IConsole[] consoles) {
 				if ("GradleConsole".equals(consoleClass.getSimpleName())) {
 					String consoleName = console.getName();
 
-					if (consoleName.contains("watch")) {
+					if (consoleName.startsWith(LiferayGradleWorkspaceProject.WATCH_LAUNCH_CONFIGURATION_NAME)) {
```
Comment (gamerson) [inline tools/plugins/com.liferay.ide.gradle.ui/src/com/liferay/ide/gradle/ui/RemoveOldWatchConsoleListener.java]: This constant shouldn't  be in this class.  The "RemoveOldWatchConsoleListener" class shouldn't know anything about "LiferayGradleWorkspaceProject".  So lets refactor this to a constants class.

File: tools/plugins/com.liferay.ide.gradle.core/src/com/liferay/ide/gradle/core/GradleUtil.java (hunk 1 above)
Comment (Camork) [reply tools/plugins/com.liferay.ide.gradle.core/src/com/liferay/ide/gradle/core/GradleUtil.java]: done

**Comment by gamerson** [comment]
Pull request submitted to https://github.com/gamerson/liferay-ide/pull/3692. [See changes here.](https://github.com/gamerson/liferay-ide/compare/13bab05b5~2...13bab05b5) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Pull request closed without merging**